### ISSUE TITLE
import: validate mime types

### ISSUE THREAD

**Issue by targos** [cheminfo/rest-on-couch#160]
We can use https://www.npmjs.com/package/mime-types

**Comment by stropitek** [comment]
If we do this we will have to extend it with our custom mime types.

I don't think jcamp files are in the specification so they won't be in the library by default

**Comment by targos** [comment]
Definitely. We need to make sure it works with custom mime types. I'd like at least to verify that the string has the correct syntax.

maybe we should use https://www.npmjs.com/package/mime
